srcpkgs/booster/files/kernel-hook-postrm

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# Kernel post-remove hook for booster.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+rm -f boot/initramfs-${VERSION}.img
```

Comment:
@ahesford I think this is a great opportunity to start implementing your initramfs chooser stuff, in whatever format.